src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -15,8 +15,9 @@
 
 import liquibase.database.AbstractJdbcDatabase;
 import liquibase.database.DatabaseConnection;
+import liquibase.structure.core.Column;
 
-public class CloudSpanner extends AbstractJdbcDatabase {
+public class CloudSpanner extends AbstractJdbcDatabase implements ICloudSpanner {
```

Comment:
Adding this interface makes it a lot easier to create proxies of the Cloud Spanner database. This can be very useful when a change type only needs very small adjustments from the standard generator. Instead of copy-pasting the code from the standard generator into the Spanner specific generator, a proxy object can be used to only override specific behavior at that moment. See the new `AddColumnGeneratorSpanner` for an example, where the only change is that we need it to generate `ALTER TABLE ADD COLUMN columnName` instead of `ALTER TABLE ADD columnName`.